src/Octoshift/Models/ConsoleWriter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+namespace OctoshiftCLI.Models
+{
+    public static class ConsoleWriter
+    {
+        public static async void OutputLogUrl(GithubApi api, string org, string repo, bool wait = true)
+        {
+            if (api != null && wait)
+            {
+                var url = await api.GetMigrationLogUrl(org, repo);
+
+                if (!string.IsNullOrEmpty(url))
+                {
+                    Console.WriteLine($"Migration log available at: {url}");
+                }
+            }
+        }
+    }
+}
+
```

Comment:
I know this PR is in draft mode, but though my early feedback may save you some time. So two things here:
1. There is no need to create a console logger, the `OctoshfitLogger` is the one and only place that handles all logs. All you need to do for this ticket is one the migration is finished, get the migration log url and write an Info log with the migration log url --> `_log.LogInformation(...)`.
2. As a bonus feedback, generally speaking whenever there is a need for a service, we create a service class and then inject it. Directly calling a static method, etc. is a code smell in many situations and at the very least it makes it vary hard and close to impossible to unit test methods.

Reply:
Noted! I misread when I went through the class and only thought it wrote to console on errors! Thanks for the tip :)